controllers/issuer_controller.go

```diff
@@ -36,6 +36,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
 
 	tcsapi "github.com/intel/trusted-certificate-issuer/api/v1alpha1"
```

Comment:
We are using now two versions of the same API in this file (v1alpha1 as tcsapi, v1alpha2 as v1alpha2). I don't have much experience about Kubernetes API versioning, but do you think this is a good idea or not?

Reply:
Yes, you are right. I also had the same question while writing this code. I was not sure if we move other Issuer APIs to v1alpha2 without any changes. But this might get solved as i am planning to move them to v1alpha2 with a few additions (choose TAC plugin/key server, default duration for signed certificate) to issuer API in upcoming PRs to support multiple key servers.